src/stories/Interactive/Code/CodeBlock.stories.tsx

```diff
@@ -55,3 +59,42 @@ InsideAccordionWithFileName.args = {
   filename: 'Example File Name',
   filenameColor: '#00ff00',
 };
+
+/*
+ * See https://storybook.js.org/docs/react/writing-stories/play-function#working-with-the-canvas
+ * to learn more about using the canvasElement to query the DOM.
+ */
+export const CodeBlockInteractions = Template.bind({});
+const testString = uuidv4();
+const fileName = 'Name 1';
+CodeBlockInteractions.args = {
+  children: testString,
+  filename: fileName,
+};
+CodeBlockInteractions.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  // 👇 Assert DOM structure.
+  await delay(20);
+  await expect(canvas.getByText(fileName)).toBeInTheDOM();
+  await expect(canvas.getByText(testString)).toBeInTheDOM();
+  await expect(canvas.getByText('Copy')).toBeInTheDOM();
+  await expect(canvas.getByText(testString)).toBeInTheDOM;
+  await expect(canvas.getByText('Copy')).not.toBeVisible();
+
+  // 👇 Simulate copy to clipboard.
+  await userEvent.click(canvas.getByText('Copy'));
+  // 👇 Assert DOM structure.
+  await delay(20);
+  await expect(canvas.getByText('Copied')).toBeVisible();
+  await expect(canvas.getByText('Copied')).toBeInTheDOM();
+
+  // 👇 Assert if copied to clipboard.
+  console.log(canvas.getByText('Copied'));
```

Comment:
Is this console log supposed to be here?
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/55263191/220215210-04a1425f-ee45-4ca3-b199-76065d201ede.png">

Reply:
good catch 👍, it's a leftover from dev

Reply:
It's removed 😉